test/end-to-end/weekendConfig/weekend_config.spec.js

```diff
@@ -18,14 +18,6 @@ describe('Weekend Configuration Management', () => {
     await page.update(newWeekendConfigLabel, updateWeekendConfigLabel);
   });
 
-  it('successfully set week days in weekend configuration', async () => {
-    await page.setWeekendConfig(updateWeekendConfigLabel);
```

Comment:
Is there somewhere that you can remove these functions?  I don't think they will ever be used again.